client/components/boxes/show/_index.html

```diff
@@ -177,7 +177,7 @@ <h3 translate>Walled Gardens</h3>
     <div heartbeat-chart height="100" target="heartbeats-0" mac="{{box.mac}}" id='heartbeats-0'></div>
   </div>
   <div flex flex-gt-sm="30" layout="column" ng-if="!fs">
-    <md-card ing-if="box.state != 'new'">
+    <md-card ng-if="box.state != 'new'">
```

Comment:
@mbecks I need to check we haven't added this for a good reason before committing.